### ISSUE TITLE
[flow] Named import from module `redux` (This module has no named export called `Store`.)

### ISSUE THREAD

**Issue by alexicum** [apollographql/react-apollo#823]
## Steps to Reproduce
1. src/types/index.js:
`import type { Store as ReduxStore, Dispatch as ReduxDispatch } from 'redux';
`
2. .flowconfig:
```
...
[libs]
./node_modules/react-apollo/lib/index.js.flow
./node_modules/apollo-client/lib/index.js.flow
...
```
3. npm run flow

### Buggy Behavior

```
src/types/index.js:3
  3: import type { Reducer, Store as ReduxStore, Dispatch as ReduxDispatch } from 'redux';
                            ^^^^^ Named import from module `redux`. This module has no named export called `Store`.

```

Works without react-apollo in libs
#./node_modules/react-apollo/lib/index.js.flow

react-apollo index.js.flow contains:
`import type { Store } from "redux";` 

commenting it also helps.

### Version
- apollo-client@1.7.0
- react-apollo@1.4.3

**Comment by jbaxleyiii** [comment]
@alexicum hmm, can you verify you have types for redux installed? Maybe try using `flow-typed` to install them?

**Comment by alexicum** [comment]
@jbaxleyiii, yes. 

`flow-typed install redux@3.x.x`

```
.\flow-typed\npm\redux_v3.x.x.js 
// flow-typed signature: 76c3dfb5a40ce169d5c08fe80dbd029a
// flow-typed version: 37b1c6d953/redux_v3.x.x/flow_>=v0.33.x 
```

.flowconfig:

```
[include]

[ignore]
<PROJECT_ROOT>/node_modules/[^a].* <PROJECT_ROOT>/node_modules/a[^p].* <PROJECT_ROOT>/node_modules/ap[^o].* <PROJECT_ROOT>/node_modules/apo[^l].* <PROJECT_ROOT>/node_modules/apol[^l].* <PROJECT_ROOT>/node_modules/apoll[^o].* <PROJECT_ROOT>/node_modules/apollo[^\-].* <PROJECT_ROOT>/node_modules/apollo-[^c].* <PROJECT_ROOT>/node_modules/apollo-c[^l].* <PROJECT_ROOT>/node_modules/apollo-cl[^i].* <PROJECT_ROOT>/node_modules/apollo-cli[^e].* <PROJECT_ROOT>/node_modules/apollo-clie[^n].* <PROJECT_ROOT>/node_modules/apollo-clien[^t].* <PROJECT_ROOT>/node_modules/apollo-client.+

[libs]
./node_modules/react-apollo/lib/index.js.flow

[options]

# https://github.com/facebook/flow/issues/2852
module.name_mapper='.*\.gql' -> '<PROJECT_ROOT>/schema/gqlStub.js'
suppress_comment=\\(.\\|\n\\)*\\$ExpectError
```

**Comment by alexicum** [comment]
Fixed in react-apollo@1.4.8
